.circleci/config.yml

```diff
@@ -62,6 +62,77 @@ jobs:
           key: machine-controller-{{ .Revision }}
           paths:
             - machine-controller
+  verify-tool:
+    docker:
+      - image: docker:stable
+    working_directory: /go/src/github.com/kubermatic/machine-controller
+    steps:
+      - restore_cache:
+          key: repo-{{ .Environment.CIRCLE_SHA1 }}
+      - restore_cache:
+          key: vendor-{{ checksum "Gopkg.lock" }}
+      - setup_remote_docker
+      - run: apk update && apk add make bash git
+      - run: |
+          set -e
+          docker create -v /go/src/github.com/kubermatic/machine-controller --name vol-container alpine /bin/true
+          docker cp . vol-container:/go/src/github.com/kubermatic/machine-controller/
+          export USE_VOLUME_CONTAINER=true && cd test/tools/verify && make verify
+          docker cp vol-container:/go/src/github.com/kubermatic/machine-controller/test/tools/verify/verify .
```

Comment:
Can you quickly explain why we do this instead of a a simple `docker build` ?

Reply:
We could do a docker build but then we would have to push it on every build